board/app/helpers/validation_helper.php

```diff
@@ -11,4 +11,13 @@ function is_logged_in(){
      return isset($_SESSION['user_id']);
 }
 
-?> 
+function validate_username($username)
+{
+	return preg_match('/^[a-zA-Z0-9._-]+$/', $username);
+}
+
+function validate_name($string)
+{
+	return preg_match('/^[a-zA-Z -]+$/', $string);
```

Comment:
You can make use of PHP built-in functions instead of creating helpers :blush: 
http://php.net/manual/en/function.ctype-alpha.php
http://php.net/manual/en/function.ctype-alnum.php